client/src/components/layout/IndexPage.js

```diff
@@ -64,8 +65,8 @@ const IndexPage = (props) => {
     event.preventDefault()
     const validationErrors = validateInput(player)
```

Comment:
Could potentially move this `validateInput` to a service object to free up some space in this component